### ISSUE TITLE
`Fira Sans` and `Droid Sans` fonts are recognised as duplicate because of word `Sans` in them

### ISSUE THREAD

**Issue by VBagmut** [biomejs/biome#6064]
### Environment information

```bash
-
```

### Rule name

-

### Playground link

-

### Expected result

The whole font name should be checked, not just a substring. To reproduce the issue - run https://create.t3.gg/ and select biome as a linter.

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by dyc3** [comment]
Thanks for the report. Would you mind updating the issue with the rule name and a playground?

**Comment by VBagmut** [comment]
the rule name is `lint/suspicious/noDuplicateFontNames`

**Comment by ematipico** [comment]
Can you please **at least** pasting the CSS?

**Comment by bitclaw** [comment]
@ematipico I was able to reproduce it [here](https://github.com/bitclaw/biome-repro-6064) using `pnpm create @biomejs/biome-reproduction`. 

I get the same error as @VBagmut 

```shell
/biome-repro-6064$ pnpm run lint

> biome-repro@0.0.0 lint /biome-repro-6064
> biome lint

src/styles/globals.css:12:9 lint/suspicious/noDuplicateFontNames ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Duplicate font names are redundant and unnecessary: Sans

    10 │                Cantarell,
    11 │                Fira Sans,
  > 12 │                Droid Sans,
       │                      ^^^^
    13 │                Helvetica Neue,
    14 │                sans-serif;

  ℹ Remove duplicate font names within the property


Checked 4 files in 1845µs. No fixes applied.
Found 1 error.
lint ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Some errors were emitted while running checks.


 ELIFECYCLE  Command failed with exit code 1.
```
